ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +35,24 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
```

Comment:
@kwaegema what code is available in `ncm-lib-blockdevices` that does
* check mounted filesystems status (which ones are mounted and should be mounted)
* check if path is below supposed-t-be-mounted filesystem